### ISSUE TITLE
Comments no longer recycled

### ISSUE THREAD

**Issue by gregsdennis** [EnricoMi/publish-unit-test-result-action#463]
I'm not sure if this is a bug that was introduced here or if the GitHub API changed, but it seems comments are no longer being recycled.

Current PR (not recycled):
- https://github.com/gregsdennis/json-everything/pull/466
- [build log](https://github.com/gregsdennis/json-everything/actions/runs/5160208921/jobs/9296137119)
Past PR (recycled, 2w ago):
- https://github.com/gregsdennis/json-everything/pull/456
- [build log](https://github.com/gregsdennis/json-everything/actions/runs/5029962152/jobs/9022076608)

**Comment by EnricoMi** [comment]
Thanks for reporting, I haven't had a chance to fully test this new feature since it arrived in master. I will take a look!

It is not recommended to use the action from master. Please move to `@v2`, which is always the latest released version.

Plus, running on Ubuntu, the non-composite version starts much quicker than the `composite` version.

    uses: EnricoMi/publish-unit-test-result-action@v2

**Comment by gregsdennis** [comment]
Yeah, I think I switched it to master after you had me try out a branch you were working on.  I've updated back to `v2`.  Looks good now.  Thanks.

If you want to keep this open to track your `master` issue, feel free.  Otherwise also feel free to close it.

**Comment by EnricoMi** [comment]
@dev-hanz-ops this has been fixed, please update https://github.com/orgs/community/discussions/24616#discussioncomment-5607870.

**Comment by dev-hanz-ops** [comment]
nice fix. I'll update my comment once it's released in `v2`, please ping me again once done
